modules/platform/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/operations/coldstorage/MoveToColdStorageTest.java

```diff
@@ -80,6 +81,29 @@ public void shouldMoveToColdStorage() throws OperationException, IOException {
         moveContentToColdStorage(session, documentModel);
     }
 
+    @Test
+    public void shouldMoveDocsToColdStorage() throws OperationException, IOException {
+        // with regular user with "WriteColdStorage" permission
+        ACE[] aces = { new ACE("linda", SecurityConstants.READ, true), //
+                new ACE("linda", SecurityConstants.WRITE, true), //
+                new ACE("linda", SecurityConstants.WRITE_COLD_STORAGE, true) };
+
+        List<DocumentModel> documents = List.of(createFileDocument(session, true, "MyFile1", aces), //
+                createFileDocument(session, true, "MyFile2", aces), //
+                createFileDocument(session, true, "MyFile3", aces));
+
+        try (CloseableCoreSession userSession = CoreInstance.openCoreSession(session.getRepositoryName(), "linda")) {
+            moveContentToColdStorage(userSession, documents);
+        }
+
+        // with Administrator
+        documents = List.of(createFileDocument(session, true, "MyFile4", aces), //
+                createFileDocument(session, true, "MyFile5", aces), //
+                createFileDocument(session, true, "MyFile6", aces));
```

Comment:
Is this expected to create the documents with "linda" `aces`?

Reply:
good point thanks, wrong copy / past